lib/templates.js

```diff
@@ -158,14 +153,8 @@ function _getEventTitle(event) {
  */
 const fCreateGroupsTemplateForEvent = (event, eventDefinition, groupIds) => {
   const ordExtensions = fReadORDExtensions(eventDefinition);
-
-  let fullyQualifiedEventName = event;
-  if (!event.includes(global.capNamespace)) {
-    fullyQualifiedEventName = global.capNamespace + "." + event;
-  }
-
   let groupId = _getGroupID(
-    fullyQualifiedEventName,
+    event,
```

Comment:
Same here, event is too general. I notice it comes from previous code. I would like give it meaningful name as well.